src/main/java/edu/hawaii/its/api/controller/GroupingsRestController.java

```diff
@@ -644,6 +646,19 @@ public ResponseEntity<String> getAsyncJobResult(Principal principal, @PathVariab
         return httpRequestService.makeApiRequest(principal.getName(), uri, HttpMethod.GET);
     }
 
+    /**
+     * Get the list of active announcements to display.
+     */
+    @GetMapping(value = "/announcements/active")
+    public ResponseEntity<String> getAnnouncements() {
+        if (!API_HANDSHAKE_ENABLED) {
+            logger.info("API Handshake is not enabled.");
+            return new ResponseEntity<>(JsonUtil.asJson(""), HttpStatus.OK);
+        }
```

Comment:
Is this if statement needed? Also add the logger.info("Entered REST getAnnouncements...") to this.

Reply:
Took out those lines and added the log, thank you.